common/config/config.go

```diff
@@ -62,6 +62,34 @@ type (
 		DomainDefaults DomainDefaults `yaml:"domainDefaults"`
 		// Blobstore is the config for setting up blobstore
 		Blobstore Blobstore `yaml:"blobstore"`
+		// Authorization is the config for setting up authorization
+		Authorization Authorization `yaml:authorization`
+	}
+
+	Authorization struct {
+		OAuthAuthorizer OAuthAuthorizer `yaml:"oauthAuthorizer"`
+		NoopAuthorizer  NoopAuthorizer  `yaml:"noopAuthorizer"`
+	}
+
+	NoopAuthorizer struct {
+		Enable bool `yaml:"enable"`
+	}
+
+	OAuthAuthorizer struct {
+		Enable bool `yaml:"enable"`
+		// Credentials to verify/create the JWT
+		JwtCredentials JwtCredentials `yaml:"jwtCredentials"`
+		// TTL: How long a JWT can be valid since timeAtIssue(iat)
+		JwtTTL int64 `yaml:"jwtTTL"`
```

Comment:
Sorry I updated the proposal -- we need to change this to MaxTTL and reject requests when claim.TTL is larger than it.